source/geometries/Next100Vessel.h

```diff
@@ -96,6 +101,14 @@ namespace nexus {
     gate_tp_distance_ = distance;
   }
 
+  inline void Next100Vessel::SetGateZpos(G4double pos){
+    gate_z_pos_ = pos;
+  }
+
+  inline G4double Next100Vessel::GetGateZpos(){
```

Comment:
Because the position of the gate with respect to the center of the gas (which is used everywhere else in the code) is calculated in the `Next100Vessel` class, therefore we need a function to get it from that class only.